modules/ossm-install-ossm-operator.adoc

```diff
@@ -14,30 +14,27 @@ To install {SMProductName}, install following Operators in this order. Repeat th
 * Kiali
 * {SMProductName}
 
+[NOTE]
+====
+If you have already installed the OpenShift Elasticsearch Operator as part of OpenShift Logging, you do not need to install the OpenShift Elasticsearch Operator again. The {JaegerName} Operator will create the Elasticsearch instance using the installed OpenShift Elasticsearch Operator.
+====
+
 .Procedure
 
-. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
 
 . In the {product-title} web console, click *Operators* -> *OperatorHub*.
 
 . Type the name of the Operator into the filter box and select the Red Hat version of the Operator. Community versions of the Operators are not supported.
 
 . Click *Install*.
-+
-[NOTE]
-====
-If you have already installed the OpenShift Elasticsearch Operator as part of OpenShift
-Logging, you do not need to install the OpenShift Elasticsearch Operator again. The
-{JaegerName} Operator will create the Elasticsearch instance using the installed OpenShift
-Elasticsearch Operator.
-====
 
-. On the *Install Operator* page, select installation options.
-.. For the OpenShift Elasticsearch Operator, in the *Update Channel* section, select *stable-5.x*.
-.. For the {JaegerName}, Kiali, and {SMProductName} Operators, accept the defaults.
-+
-The Kiali and {SMProductName} Operators are installed in the `openshift-operators` namespace. The {JaegerName} is installed in the `openshift-distributed-tracing` namespace. The OpenShift Elasticsearch Operator is installed in the `openshift-operators-redhat` namespace.
+. On the *Install Operator* page for each Operator, accept the defaults.
```

Comment:
accept _what_ defaults? Permissions? Namespaces? Can we provide more context?

Reply:
This is one of those put it in, take it out situations.  We used to have four Operator installation modules.  Neal decided that it was repetitive and that we only needed a single module for how to install the four Operators.  But since the Operator settings are slightly different from each other,  this step is vague on purpose.